week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,9 +42,23 @@ The function must:
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+    let removedArr = arr;
+    removedArr.splice(index, 1);
+    return removedArr; // complete this statement
```

Comment:
If you read the instructions, you will see it says Not change the original array
try to search for Splice destructive nature
the original array will be affected, Slice can be a possible way to fix it